.azure-pipelines/client.yml

```diff
@@ -6,28 +6,25 @@
 trigger:
 - master
 
+variables:
+  PythonVersion: '3.6'
+
 jobs:
 - job: 'Build_Target_Packages'
```

Comment:
Can you please update this job to call it "Build" as we added in the guidelines?